### PR TITLE
Feat/rebrand

### PR DESCRIPTION
**Pull request by KillerX**
Rebranding images:

Known issues:

* Android Splash

### CODE REVIEW

**Review by andreasgan — CHANGES_REQUESTED** [review]
for the ios splash, it looks like all 3 images are the same.
they also include a background. can you just use the logo alone and use a background on the .storyboard instead?
Storyboards have a visual editor in xcode btw.

**Comment by KillerX** [comment]
@andreasgan I removed the splash images in iOs, and added the proper 2 layer icon to Andorid, so various shapes look ok now:

image

File: ios/Runner/Info-Debug-prod.plist [hunk 1]
```diff
@@ -2,8 +2,6 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
-	<key>ITSAppUsesNonExemptEncryption</key>
```
Comment (andreasgan) [inline ios/Runner/Info-Debug-prod.plist:5]: curious why you removed this

**Review by andreasgan — APPROVED** [review]

File: ios/Runner/Info-Debug-prod.plist (hunk 1 above)
Comment (KillerX) [reply ios/Runner/Info-Debug-prod.plist:5]: Xcode just moved it down below to line 50. Didn't manually edit this file

**Pull request merged**